brave/brave-core
# Enable sidebar on right side

Resolves https://github.com/brave/brave-browser/issues/25332
Resolves https://github.com/brave/brave-browser/issues/25756

Layout w/o vertical tab:
<img width="1345" alt="Screen Shot 2022-10-04 at 11 49 15 AM" src="https://user-images.githubusercontent.com/6786187/193723954-cbe9a570-4255-44f4-887d-d88b24d256b0.png">
<img width="1337" alt="Screen Shot 2022-10-04 at 11 49 29 AM" src="https://user-images.githubusercontent.com/6786187/193723958-ffee4762-343d-4307-a96e-4f0d438e0006.png">

Layout with vertical tab:
<img width="1317" alt="Screen Shot 2022-10-04 at 11 47 26 AM" src="https://user-images.githubusercontent.com/6786187/193723799-71850961-6b1a-499e-a087-93ce8d57f0e6.png">
<img width="1318" alt="Screen Shot 2022-10-04 at 11 47 39 AM" src="https://user-images.githubusercontent.com/6786187/193723802-98e0cf17-a90a-4b41-9fa9-b010d9614234.png">


## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

`SidebarBrowserTestWithVerticalTabs.SidebarRightSideTest`

1. Launch with clean profile
2. Load brave://settings/appearance and check `Side panel` option is visible
3. Check default is `Show on left`
4. Select `Show on right` and sidebar is visible on right side
5. Open/close any sidebar items and checks works as expected
6. Enable vertical tab via brave://flags and re-launch
7. Check sidebar is located properly with vertical tab
